### ISSUE TITLE
Beautifying markdown with prettier crashes

### ISSUE THREAD

**Issue by kachkaev** [Glavin001/atom-beautify#2070]
# Description

Looks like Prettier is trying to format markdown files with some other syntax, because no matter what markdown I input, it throws an exception. A popup that usually shows for long-running async beautifiers hangs forever.

# Actual Output

![screen shot 2018-03-06 at 15 54 28](https://user-images.githubusercontent.com/608862/37042712-f012a24e-2156-11e8-96fa-1ebd7d6b8485.png)

# Steps to Reproduce

1. Create and save a markdown file with any simple contents like:

    ```md
    # abc
    
    test
    ```

1. Set prettier as your default beautifier for markdown, e.g:

```cson
"*":
  "atom-beautify":
    general:
      beautifyEntireFileOnSave: false
    markdown:
      default_beautifier: "Prettier"
```

2. Run command `Atom Beautify: Beautify Editor`

# Debug

The command does not work and I see this error in Atom's console:

```
/Users/ak/.atom/packages/atom-beautify/node_modules/prettier/parser-babylon.js:1 Uncaught (in promise) SyntaxError: Unexpected character '#' (1:1)
> 1 | # abc
    | ^
  2 | 
  3 | test
  4 | 
    at createError (/Users/ak/.atom/packages/atom-beautify/node_modules/prettier/parser-babylon.js:1)
    at Object.parse (/Users/ak/.atom/packages/atom-beautify/node_modules/prettier/parser-babylon.js:1)
    at Object.parse$4 [as parse] (/Users/ak/.atom/packages/atom-beautify/node_modules/prettier/index.js:23124)
    at formatWithCursor (/Users/ak/.atom/packages/atom-beautify/node_modules/prettier/index.js:32482)
    at format (/Users/ak/.atom/packages/atom-beautify/node_modules/prettier/index.js:32527)
    at Object.format (/Users/ak/.atom/packages/atom-beautify/node_modules/prettier/index.js:32787)
    at prettier.coffee [sm]:10
```

<img width="470" alt="screen shot 2018-03-06 at 16 02 54" src="https://user-images.githubusercontent.com/608862/37043154-da41fb8a-2157-11e8-8187-12e45b56d5c6.png">

# Checklist

I have:
- [x] Tried uninstalling and reinstalling Atom Beautify to ensure it installed properly
- [x] Reloaded (or restarted) Atom to ensure it is not a caching issue
- [x] Searched through existing Atom Beautify Issues at https://github.com/Glavin001/atom-beautify/issues
so I know this is not a duplicate issue
- [x] Filled out the Input, Expected, and Actual sections above or have edited/removed them in a way that fully describes the issue.
- [x] Generated debugging information by executing `Atom Beautify: Help Debug Editor` command in Atom and added link for `debug.md` Gist to this issue

cc @szeck87

**Comment by szeck87** [comment]
That's not good, I have a fix in #2071.

**Comment by Glavin001** [comment]
Thanks for reporting, @kachkaev !

Prettier is not the default Markdown beautifier, right? This only happens for users who manually switched to Prettier?

**Comment by kachkaev** [comment]
Yep.

**Comment by Glavin001** [comment]
Published to v0.32.2

**Comment by muppetjones** [comment]
Getting the same error in 0.33.4
```
Uncaught (in promise) SyntaxError: Unexpected character '#' (1:2)
> 1 | # Variant
    |  ^
  2 | 
  3 | Format Version v1.0.0
  4 | 
    at t (/Users/stephen.bush/.atom/packages/atom-beautify/node_modules/prettier/parser-babylon.js:1:419)
    at Object.d [as parse] (/Users/stephen.bush/.atom/packages/atom-beautify/node_modules/prettier/parser-babylon.js:1:188061)
    at Object.parse$2 [as parse] (/Users/stephen.bush/.atom/packages/atom-beautify/node_modules/prettier/index.js:7138:19)
    at coreFormat (/Users/stephen.bush/.atom/packages/atom-beautify/node_modules/prettier/index.js:10398:23)
    at format (/Users/stephen.bush/.atom/packages/atom-beautify/node_modules/prettier/index.js:10570:16)
    at formatWithCursor (/Users/stephen.bush/.atom/packages/atom-beautify/node_modules/prettier/index.js:10582:12)
    at /Users/stephen.bush/.atom/packages/atom-beautify/node_modules/prettier/index.js:34924:15
    at Object.format (/Users/stephen.bush/.atom/packages/atom-beautify/node_modules/prettier/index.js:34943:12)
    at /Users/stephen.bush/.atom/packages/atom-beautify/src/beautifiers/prettier.coffee:42:29
    at <anonymous>
```
